### PR TITLE
refactor: Update editPerson

### PR DESCRIPTION
**Pull request by m1oojv**
## What type of PR is this? (check all applicable)

-   [x] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [ ] Documentation Update

## Description

-   Update the command `edit` to `editPerson`
-   Update related testcases
-   Remove unwanted files 

## Related Tickets & Documents

-   Related Issue #41 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [x] Yes
-   [ ] No, and this is why: _please replace this line with details on why tests
        have not been included_
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditPersonCommandParser.java [hunk 1]
```diff
@@ -14,22 +14,22 @@
...
 import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
+import seedu.address.logic.commands.EditPersonCommand;
+import seedu.address.logic.commands.EditPersonCommand.EditPersonDescriptor;
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.tag.Tag;
 
 /**
  * Parses input arguments and creates a new EditCommand object
```
Comment (HugeNoob) [inline src/main/java/seedu/address/logic/parser/EditPersonCommandParser.java:23]:
```suggestion
 * Parses input arguments and creates a new EditPersonCommand object
```
Would be good if you could rename the commands in comments as well.

File: src/main/java/seedu/address/logic/parser/EditPersonCommandParser.java [hunk 2]
```diff
@@ -14,22 +14,22 @@
...
  */
-public class EditCommandParser implements Parser<EditCommand> {
+public class EditPersonCommandParser implements Parser<EditPersonCommand> {
 
     /**
      * Parses the given {@code String} of arguments in the context of the EditCommand
      * and returns an EditCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
```
Comment (HugeNoob) [inline src/main/java/seedu/address/logic/parser/EditPersonCommandParser.java:30]:
```suggestion
     * Parses the given {@code String} of arguments in the context of the EditPersonCommand
     * and returns an EditPersonCommand object for execution.
     * @throws ParseException if the user input does not conform the expected format
```
Same for here and similar places (if any).

File: src/test/java/seedu/address/logic/commands/EditPersonCommandTest.java [hunk 3]
```diff
@@ -72,10 +74,11 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
 
     @Test
     public void execute_noFieldSpecifiedUnfilteredList_success() {
-        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
+        EditPersonCommand editCommand = new EditPersonCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
```
Comment (HugeNoob) [inline src/test/java/seedu/address/logic/commands/EditPersonCommandTest.java:77]:
```suggestion
        EditPersonCommand editPersonCommand = new EditPersonCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
```
Do you think it's worth renaming these variables as well? I know it doesn't change the functionality, but perhaps it's more explicit. Same for everywhere else in this file. Wdyt?

**Review by HugeNoob — CHANGES_REQUESTED** [review]
Mostly LGTM. Some nits over renaming in comments and variables. Let me know if you feel they are unnecessary changes.

**Review by HugeNoob — APPROVED** [review]

**Review by pra-navi — APPROVED** [review]
LGTM, thanks for the changes

**Pull request merged**